bower.json

```diff
@@ -18,7 +18,18 @@
     "shim"
   ],
   "license": "BSD-3-Clause",
-  "ignore": [],
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "src",
```

Comment:
Why send down gulpfile.js and package.json in the bower directory? What value do they provide to the consuming module?